service/SpaceCenter/src/Services/Parts/Experiment.cs

```diff
@@ -146,7 +146,20 @@ public void Reset ()
         /// </summary>
         [KRPCProperty]
         public IList<ScienceData> Data {
-            get { return experiment.GetData ().Select (data => new ScienceData (experiment, data)).ToList (); }
+            get { return experiment.GetData().Select(data => new ScienceData(experiment, data)).ToList (); }
+        }
+
+        /// <summary>
+        /// Determines if the experiment is available given the situationMask and requireAtmosphere.
```

Comment:
What are situationMask and requireAtmosphere? Does this property just return whther the experiment is available given the current conditions?